README.md

```diff
@@ -1,2 +1,40 @@
 # terraform-provider-qualys
-A Terraform provider for Qualys resources
+
+A Terraform provider for Qualys CloudView Resources
+
+## References
+
+Based on original work from here - [https://code.stanford.edu/xuwang/terraform-provider-qualys]
+
+Qualys CloudView API Docs - [https://qualysguard.qg2.apps.qualys.com/cloudview-api/swagger-ui.html#/]
```

Comment:
It would be nice to generate docs based on the resource schemas using `tfplugindocs`. You can see how that's done in another provider: https://github.com/form3tech-oss/terraform-provider-grafanacloud/blob/master/Makefile#L53-L54.

That goes hand in hand with the `Description` field that we can set in the various schema resources, e.g. https://github.com/form3tech-oss/terraform-provider-grafanacloud/blob/master/grafanacloud/resource_portal_api_key.go.

We could then link to the docs and make the available resources more discoverable.

Reply:
Good idea, pushed some changes adding docs. This needed an update to v2 of the terraform plugin SDK, which we probably should have been using anyway.